assets/aws/files/bin/teleport-generate-config

```diff
@@ -275,10 +275,7 @@ auth_service:
     region: ${EC2_REGION}
   app_publisher_config:
     enabled: true
-    region: ${EC2_REGION}
-  kms_encryption_config:
-    region: ${EC2_REGION}
-    kms_key_id: ${KMS_KEY_ID}   
+    region: ${EC2_REGION}  
```

Comment:
do we still need the region?

Reply:
Diff is showing at wrong place, region is used for app_publisher_config